.github/workflows/ci.yml

```diff
@@ -19,17 +19,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python 3.9
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4.0.0
```

Comment:
We might consider just `v4` so that any incremental/nonbreaking updates or fixes don't necessarily need code changes to enable/fix -- for example, the built-in cacheing support in `v2.3.0`. (For more examples of fixes and enhancements refer to the [minor and patch release notes](https://github.com/actions/setup-python/releases).)

Reply:
Fixed, thanks!